engine/releases/supabaseProvider.ts

```diff
@@ -134,10 +138,11 @@ export const newSupabase = (
         notify();
       }, (_status, err) => {
         if (err) {
-          console.error(
-            "error when trying to subscribe to release changes falling back to background updates",
-            err,
-          );
+          const errMsg =
+            `error when trying to subscribe to release changes falling back to background updates, ${err}`;
+          logger.error(errMsg);
+          console.error(errMsg);
```

Comment:
Begs the question if we need this

Reply:
yes, because somehow the logger.error interface is different from the console.error